learning_journal/views/default.py

```diff
@@ -53,20 +52,20 @@ def edit(request):
             creation_date = time.strftime("%m/%d/%Y")
```

Comment:
Can't comment above this line, but in your `detail` view why not use `query.get(request.matchdict['id'])` like you did in this view?